libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -96,6 +97,18 @@ public void updateSessionIdRotationInterval(int hour) {
     }
   }
 
+  public boolean isDebugLoggingEnabled() {
+    return debugLoggingEnabled;
+  }
+
+  public void setDebugLoggingEnabled(boolean debugLoggingEnabled) {
+    this.debugLoggingEnabled = debugLoggingEnabled;
```

Comment:
We're assigning `debugLoggingEnabled` even if `telemetryClient` is `null` and we don't want that.

In any case we should consider adding `debugLoggingEnabled` setting within `TelemetryClientSettings` 👇 and update it here somehow.